### PR TITLE
[BUGFIX] Fix no-docs input option

### LINKED ISSUE #38: set-version no-docs option looks wrong
The input type of the no-docs option should imho be NO_VALUE see https://github.com/TYPO3/tailor/blob/2b4fa52ea9178c9c606018bba62a6b2bfd36cc7a/src/Command/Extension/SetExtensionVersionCommand.php#L42

### PR DESCRIPTION
**Pull request by gilbertsoft**
Fixes #38

### CODE REVIEW

File: src/Command/Extension/SetExtensionVersionCommand.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ protected function configure(): void
             ->setDescription('Update the extensions ext_emconf.php version to a specific version. Useful in CI environments')
             ->addArgument('version', InputArgument::REQUIRED, 'The version to publish, e.g. 1.2.3. Must have three digits.')
             ->addOption('path', '', InputOption::VALUE_OPTIONAL, 'Path to the extension folder', getcwd() ?: './')
-            ->addOption('no-docs', '', InputOption::VALUE_OPTIONAL, 'Disable version update in documentation settings', false);
+            ->addOption('no-docs', '', InputOption::NO_VALUE, 'Disable version update in documentation settings');
```
Comment (o-ba) [inline src/Command/Extension/SetExtensionVersionCommand.php:42]: This now leads to:

> PHP Fatal error:  Uncaught Error: Undefined class constant 'NO_VALUE'

**Review by o-ba — COMMENTED** [review]
Hi, besides the non existing constant, I thought about this again and fear this would be a breaking change for people, already using `--no-docs=1` in their workflows, because this would then fail with:

> The "--no-docs" option does not accept a value.

Additionally, the check in L:76 would need to be adjusted since `NULL` is no longer to be expected.

File: src/Command/Extension/SetExtensionVersionCommand.php (hunk 1 above)
Comment (gilbertsoft) [reply src/Command/Extension/SetExtensionVersionCommand.php:42]: Oops, it's ofc VALUE_NONE.

**Comment by gilbertsoft** [comment]
> 
> 
> Hi, besides the non existing constant, I thought about this again and fear this would be a breaking change for people, already using `--no-docs=1` in their workflows, because this would then fail with:
> 
> > The "--no-docs" option does not accept a value.
> 
> Additionally, the check in L:76 would need to be adjusted since `NULL` is no longer to be expected.

Agree, this is ofc kind of breaking. Will fix the error so it's ready to merge when the right time comes or feel free to close.

**Comment by o-ba** [comment]
> Agree, this is ofc kind of breaking. Will fix the error so it's ready to merge when the right time comes or feel free to close.

Oke I would like to postpone this for now. Thanks for the adjustments, we might bring this up again in the future.

**Pull request closed without merging**